can/packer.cc

```diff
@@ -45,24 +45,23 @@ std::vector<uint8_t> CANPacker::pack(uint32_t address, const std::vector<SignalP
 
   // set all values for all given signal/value pairs
   bool counter_set = false;
-  for (const auto& sigval : signals) {
-    auto sig_it = signal_lookup.find(std::make_pair(address, sigval.name));
-    if (sig_it == signal_lookup.end()) {
-      // TODO: do something more here. invalid flag like CANParser?
-      WARN("undefined signal %s - %d\n", sigval.name.c_str(), address);
-      continue;
```

Comment:
we should make sure to preserve the warning for specifying non-existent signals, or better yet, make it apparent you're doing something weird: https://github.com/commaai/opendbc/issues/500